File: fix.py
import os

files = [
    "D:/gwnagju-eco-tour/components/map/BusStopMarker.tsx",
    "D:/gwnagju-eco-tour/components/map/SubwayStationMarker.tsx",
    "D:/gwnagju-eco-tour/components/map/TourismMarker.tsx"
]

for f in files:
    with open(f, "r", encoding="utf-8") as file:
        content = file.read()
    content = content.replace("\\`data:image", "`data:image")
    content = content.replace("\\${", "${")
    content = content.replace("}\\`", "}`")
    with open(f, "w", encoding="utf-8") as file:
        file.write(content)
print("Done")

File: fix.js
const fs = require('fs');

function fixFile(file) {
  let content = fs.readFileSync(file, 'utf8');
  content = content.replace(/\\\data:image\/svg\+xml;utf8,\\\$\\{/g, '\data:image/svg+xml;utf8,\\\\/g, '}\');
  fs.writeFileSync(file, content);
}

fixFile('D:/gwnagju-eco-tour/components/map/BusStopMarker.tsx');
fixFile('D:/gwnagju-eco-tour/components/map/SubwayStationMarker.tsx');
fixFile('D:/gwnagju-eco-tour/components/map/TourismMarker.tsx');

console.log('Fixed syntax errors');
